ciongke/discover-team-id

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script prints the ID of a given GitHub team. You'll need to pass it your
+# username, the path to your OAuth token, the org name and the team name. It
+# will print out the team ID.
+# This will flake if there are more than 100 teams in the org.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ $# -ne 4 ]]; then
+    echo "Usage: $0 user token_file org team" >&2
+    exit 1
+fi
+
+readonly user=$1
+readonly token_file=$2
+readonly org=$3
+readonly team=$4
+
+curl -s -u "${user}:$(cat "${token_file}")" "https://api.github.com/orgs/${org}/teams?per_page=100" | jq ".[] | select(.name == \"${team}\") | .id"
```

Comment:
Then the bash one-liner won't work. I don't think this is worth solving here. The kubernetes org currently has 72 teams.